src/room.cpp

```diff
@@ -167,9 +167,20 @@ void Room::SendJoinSuccess(const Member& member) {
 
 void Room::BroadcastRoomInformation() {
     RakNet::BitStream stream;
-    stream.Write(static_cast<RakNet::MessageID>(ID_ROOM_JOIN_SUCCESS));
-
-    // TODO(Subv): Add the room and member information to the packet.
+    stream.Write(static_cast<RakNet::MessageID>(ID_ROOM_INFORMATION));
+
+    RakNet::RakString room_name = room_information.name.c_str();
+    stream.Write(room_name);
+    stream.Write(room_information.member_slots);
+
+    stream.Write(members.size());
```

Comment:
Cast this to an uint32_t.
size() is size_t which has varying length in different platforms